### PR TITLE
removing magic numbers from bookmarklet.js

### PR DESCRIPTION
**Pull request by jyoo980**

### CODE REVIEW

File: bookmarklet.js [hunk 1]
```diff
@@ -15,7 +17,7 @@ javascript:(
                         clearInterval(intervalPoll);
                         callback();
                     };
-                }, 69);
```
Comment (jyoo980) [inline bookmarklet.js:18]: magic number bad

File: bookmarklet.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 javascript:(
-    function(){
+    () => {
+        const hahaFunnySegsNumber = 69;
+        const hahaFunnyWeedNumber = 420;
         if (!document.getElementById("generate-button")) {
             console.stdlog = console.log.bind(console);
             console.logs = [];
-            console.log = function(){
+            console.log = () => {
```
Comment (jyoo980) [inline bookmarklet.js]: fat arrow function go brrrr

Comment (ianmah) [reply bookmarklet.js]: this doesnt work

**Review by ianmah — CHANGES_REQUESTED** [review]

File: bookmarklet.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 javascript:(
-    function(){
+    function() {
+        const hahaFunnySegsNumber = 69;
+        const hahaFunnyWeedNumber = 420;
         if (!document.getElementById("generate-button")) {
             console.stdlog = console.log.bind(console);
             console.logs = [];
-            console.log = function(){
+            console.log = function () {
```
Comment (ianmah) [inline bookmarklet.js]: nit: did you mean to do this?

Comment (jyoo980) [reply bookmarklet.js]: no, fixed

**Comment by verdantpeas** [comment]
:shipit:

File: bookmarklet.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
 javascript:(
-    function(){
+    function() {
+        const hahaFunnySegsNumber = 69;
+        const hahaFunnyWeedNumber = 420;
```
Comment (ianmah) [inline bookmarklet.js]:
```suggestion
        const pollInterval = 69;
        const resultWaitTime = 420;
```

**Review by ianmah — APPROVED** [review]
🚀

**Pull request merged**